### DISCUSSION TITLE
Future of Freenode IRC Channel

### DISCUSSION THREAD

**Discussion by pyrmont** [janet-lang/janet#700] [General]
@bakpakin In light of the recent controversy surrounding Freenode (see [here](https://www.theregister.com/2021/05/19/freenode_staff_resigns/)), do you have any thoughts about what to do with the #janet IRC channel? Would you prefer to stop having an official IRC channel altogether and consolidate chat on Gitter/Matrix?

Incidentally, I have registered the #janet channel on Libera as a precautionary measure to stop squatters. Am happy to turn that over if you're thinking of moving there.

**Comment by pepe** [comment]
I had the same idea. I registered #janet-lang on libera for the same reasons.

**Comment by bakpakin** [comment]
Yeah, the irc channel is honestly pretty dead and anyone that dedicated to IRC would he better off using a bridge. I will remove the irc instructions from the readme and website.

**Reply by pyrmont** [reply]
Sounds good. I'll stay in the #janet channel on Libera but have made an announcement that it's not an official discussion forum.

**Reply by subsetpark** [reply]
Should we install a Gitter bridge in that libera channel? Is that possible ?

**Reply by pyrmont** [reply]
I only did a cursory search but [Matterbridge](https://github.com/42wim/matterbridge) looks like something that can run on a remote server and echo messages on one platform to another.

My preference would be to leave them as separate, though. I think there are different cultures to IRC (pasting code to external services, plaintext messages) that are inconsistent with how people use services like Gitter/Matrix.
